README.md

```diff
@@ -1,2 +1,35 @@
-# Tic-Tac-Toe
-A game built with Ruby
+## Tic-Tac-Toe
+
+In this project we had the opportunity to build Tic Tac Toe game in Ruby. Just in case you're unfamiliar with the game, let's discuss the rules first. Tic Tac Toe is played on a 3 x 3 grid of available cells, and the goal of the game is to place three of your tokens (X or O) on the grid in a row, including diagonals.
+
+Tic Tac Toe is a game that flexes the basic concepts in programming. For example, a Tic Tac Toe program requires data structures for storing the board and conditional logic for knowing whose turn it is or if someone has won.
+
+It is a game played by two people, each trying to make a line of three X's or three O's in a boxlike figure with nine spaces. It is either won horizontally, vertically or diagonally.
+
+The first player that has a line of three X's or three O's wins the game.
+
+## How to set up in local machine
+
+Tictactoe is easier to setup. You can use Sourcetree, Git from the command line, or any client you like to clone your Git repository. These instructions show you how to clone your repository using Git from the terminal. First, from the repository, click + in the global sidebar and select Clone this repository under Get to work. Secondly, Copy the clone command (either the SSH format or the HTTPS). If you are using the SSH protocol, ensure your public key is in github and loaded on the local system to which you are cloning. Forth, From a terminal window, change to the local directory where you want to clone your repository. Paste the command you copied from Github.
+
+If the clone was successful, a new sub-directory appears on your local drive in the directory where you cloned your repository. This directory has the same name as the github repository that you cloned. The clone contains the files and metadata that Git requires to maintain the changes you make to the source files. At that point you will be ready to view enumerable files and install them locally.
```

Comment:
- how does someone clone the repo. what command should they type? include technical details
- be precise with the instructions